### PR TITLE
Poggers Report 31

### PR DESCRIPTION
**Pull request by ike709**
TODO:

- [x] Patrons list
- [x] Bully people for feedback
- [x] Address feedback

### CODE REVIEW

File: content/post/22-03-16-progress-report-31.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+{{< newsection >}}
+## Content
+{{< newsection >}}
+### Two-Parter
+There's been a long list of contributions since the last progress report, so this edition just covers January 2022. The February 2022 edition will follow in a week or so.
+
+{{< newsection >}}
+### Game Wiki
+
+Space Station 14 now has a [wiki](https://wiki.spacestation14.io/wiki/Main_Page)! It's a one-stop shop for everything you need to know in order to get that greentext.
```
Comment (moonheart08) [inline content/post/22-03-16-progress-report-31.md:23]: Perhaps mention that it is open to contributions? People can log in with their SS14 account.

Comment (ike709) [inline content/post/22-03-16-progress-report-31.md:23]:
```suggestion
Space Station 14 now has a [wiki](https://wiki.spacestation14.io/wiki/Main_Page)! It's a one-stop shop for everything you need to know in order to get that greentext, and anyone with a Space Station 14 account is able to contribute.
```

File: content/post/22-03-16-progress-report-31.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+### Energy Swords
+*Contributed by Mirino97*
+
+Swish, swish, stab. It's a sword, dude, not a fighter jet.
+
+{{< imgw "images/post/pr_31/esword" >}}
+
+{{< newsection >}}
+### Duck!
+*Contributed by Daemon*
```
Comment (ike709) [inline content/post/22-03-16-progress-report-31.md:103]:
```suggestion
*Contributed by Daemon and Mithrandalf*
```

File: content/post/22-03-16-progress-report-31.md [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+
+### Patrons
+*Aaron Mell, Acruid, Alex Evgrashin, BubblegumBlue, Charlese2, Clyybber, Delete69, Demetre Beroshvili, DrSmugleaf, E F R, ElectroJr, Fishfish458, Flipp Syder, Jack Fox, Julian Giebel, JustinTime, Júlio César Ueti, Kara D, Lamrr, Leon Friedrich, LittleBuilderJane, Matz05, Michael Phillips, Moony, Morbo, Neith, Pancake, Paul, Peptide90, Pieter-Jan Briers, Plasmaguy, Rane, Sam Weaver, ScalyChimp, ShadowCommander, T-Stalker, TaralGit, Vera Aguilera Puerto, Willhelm53, WlarusFromDaSpace, Ygg01, ZeroDayDaemon, Zymem, bloodrizer, keronshb, metalgearsloth, mirrorcult, pointer-to-null, router, sewerpig, wrexbe*
```
Comment (ike709) [inline content/post/22-03-16-progress-report-31.md:165]:
```suggestion
*Aaron Mell, Acruid, Alex Evgrashin, BubblegumBlue, Charlese2, Clyybber, Daemon, Delete69, Demetre Beroshvili, DrSmugleaf, E F R, ElectroJr, Fishfish458, Flipp Syder, Jack Fox, Julian Giebel, JustinTime, Júlio César Ueti, Kara D, Lamrr, Leon Friedrich, LittleBuilderJane, Matz05, Michael Phillips, Moony, Morbo, Neith, Pancake, Paul, Peptide90, Pieter-Jan Briers, Plasmaguy, Rane, Sam Weaver, ScalyChimp, ShadowCommander, T-Stalker, TaralGit, Vera Aguilera Puerto, Willhelm53, WlarusFromDaSpace, Ygg01, Zymem, bloodrizer, keronshb, metalgearsloth, mirrorcult, pointer-to-null, router, sewerpig, wrexbe*
```

File: content/post/22-03-16-progress-report-31.md [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+*Contributed by DrSmugleaf and mirrorcult*
+
+`Content.MapRenderer` has been added as a new project that can turn a map file and render it as an image. It's extremely useful for development, updating the wiki, and more.
+
+{{< imgw "images/post/pr_31/bagelrender" >}}
+
+{{< newsection >}}
+## Credits
+### Contributors Since Last Progress Report
+*354 commits, authored by: metalgearsloth, Leon Friedrich, Peptide90, Rane, Pancake, Kara D, ShadowCommander, Moony, Lamrr, mirrorcult, wrexbe, Pieter-Jan Briers, Vera Aguilera Puerto, LittleBuilderJane, Alex Evgrashin, Michael Phillips, Morbo, Sam Weaver, Paul Ritter, ZeroDayDaemon, Delete69, ElectroJr, Jack Fox, Júlio César Ueti, Paul, ScalyChimp, moonheart08, BubblegumBlue, Charlese2, DrSmugleaf, E F R, Fishfish458, T-Stalker, Zymem, Aaron Mell, Acruid, Clyybber, Demetre Beroshvili, Flipp Syder, Javier Guardia Fernández, Julian Giebel, JustinTime, Matz05, Neith, Plasmaguy, TaralGit, Willhelm53, WlarusFromDaSpace, Ygg01, bloodrizer, keronshb, pointer-to-null, router, sewerpig*
```
Comment (ike709) [inline content/post/22-03-16-progress-report-31.md:162]:
```suggestion
*354 commits, authored by: metalgearsloth, Leon Friedrich, Peptide90, Rane, Pancake, Kara D, ShadowCommander, Moony, Lamrr, mirrorcult, wrexbe, Pieter-Jan Briers, Vera Aguilera Puerto, LittleBuilderJane, Alex Evgrashin, Michael Phillips, Morbo, Sam Weaver, Paul Ritter, Daemon, Delete69, ElectroJr, Jack Fox, Júlio César Ueti, Paul, ScalyChimp, moonheart08, BubblegumBlue, Charlese2, DrSmugleaf, E F R, Fishfish458, T-Stalker, Zymem, Aaron Mell, Acruid, Clyybber, Demetre Beroshvili, Flipp Syder, Javier Guardia Fernández, Julian Giebel, JustinTime, Matz05, Neith, Plasmaguy, TaralGit, Willhelm53, WlarusFromDaSpace, Ygg01, bloodrizer, keronshb, pointer-to-null, router, sewerpig*
```

Comment (ike709) [inline content/post/22-03-16-progress-report-31.md]:
```suggestion
*354 commits, authored by: metalgearsloth, Peptide90, Rane, Pancake, Kara D, ShadowCommander, Moony, Lamrr, mirrorcult, wrexbe, Pieter-Jan Briers, Vera Aguilera Puerto, LittleBuilderJane, Alex Evgrashin, Michael Phillips, Morbo, Sam Weaver, Paul Ritter, ZeroDayDaemon, Delete69, ElectroJr, Jack Fox, Júlio César Ueti, Paul, ScalyChimp, moonheart08, BubblegumBlue, Charlese2, DrSmugleaf, E F R, Fishfish458, T-Stalker, Zymem, Aaron Mell, Acruid, Clyybber, Demetre Beroshvili, Flipp Syder, Javier Guardia Fernández, Julian Giebel, JustinTime, Matz05, Neith, Plasmaguy, TaralGit, Willhelm53, WlarusFromDaSpace, Ygg01, bloodrizer, keronshb, pointer-to-null, router, sewerpig*
```

File: content/post/22-03-16-progress-report-31.md [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+*Contributed by DrSmugleaf and mirrorcult*
+
+`Content.MapRenderer` has been added as a new project that can turn a map file and render it as an image. It's extremely useful for development, updating the wiki, and more.
+
+{{< imgw "images/post/pr_31/bagelrender" >}}
+
+{{< newsection >}}
+## Credits
+### Contributors Since Last Progress Report
+*354 commits, authored by: metalgearsloth, Leon Friedrich, Peptide90, Rane, Pancake, Kara D, ShadowCommander, Moony, Lamrr, mirrorcult, wrexbe, Pieter-Jan Briers, Vera Aguilera Puerto, LittleBuilderJane, Alex Evgrashin, Michael Phillips, Morbo, Sam Weaver, Paul Ritter, Daemon, Delete69, ElectroJr, Jack Fox, Júlio César Ueti, Paul, ScalyChimp, moonheart08, BubblegumBlue, Charlese2, DrSmugleaf, E F R, Fishfish458, T-Stalker, Zymem, Aaron Mell, Acruid, Clyybber, Demetre Beroshvili, Flipp Syder, Javier Guardia Fernández, Julian Giebel, JustinTime, Matz05, Neith, Plasmaguy, TaralGit, Willhelm53, WlarusFromDaSpace, Ygg01, bloodrizer, keronshb, pointer-to-null, router, sewerpig*
```
Comment (ike709) [inline content/post/22-03-16-progress-report-31.md:162]:
```suggestion
*354 commits, authored by: metalgearsloth, Peptide90, Rane, Pancake, Kara D, ShadowCommander, Moony, Lamrr, mirrorcult, wrexbe, Pieter-Jan Briers, Vera Aguilera Puerto, LittleBuilderJane, Alex Evgrashin, Michael Phillips, Morbo, Sam Weaver, Paul Ritter, Daemon, Delete69, ElectroJr, Jack Fox, Júlio César Ueti, Paul, ScalyChimp, moonheart08, BubblegumBlue, Charlese2, DrSmugleaf, E F R, Fishfish458, T-Stalker, Zymem, Aaron Mell, Acruid, Clyybber, Demetre Beroshvili, Flipp Syder, Javier Guardia Fernández, Julian Giebel, JustinTime, Matz05, Neith, Plasmaguy, TaralGit, Willhelm53, WlarusFromDaSpace, Ygg01, bloodrizer, keronshb, pointer-to-null, router, sewerpig*
```

File: content/post/22-03-16-progress-report-31.md [hunk 6]
```diff
@@ -0,0 +1,165 @@
...
+*Contributed by DrSmugleaf and mirrorcult*
+
+`Content.MapRenderer` has been added as a new project that can turn a map file and render it as an image. It's extremely useful for development, updating the wiki, and more.
+
+{{< imgw "images/post/pr_31/bagelrender" >}}
+
+{{< newsection >}}
+## Credits
+### Contributors Since Last Progress Report
+*354 commits, authored by: metalgearsloth, Peptide90, Rane, Pancake, Kara D, ShadowCommander, Moony, Lamrr, mirrorcult, wrexbe, Pieter-Jan Briers, Vera Aguilera Puerto, LittleBuilderJane, Alex Evgrashin, Michael Phillips, Morbo, Sam Weaver, Paul Ritter, Daemon, Delete69, ElectroJr, Jack Fox, Júlio César Ueti, Paul, ScalyChimp, moonheart08, BubblegumBlue, Charlese2, DrSmugleaf, E F R, Fishfish458, T-Stalker, Zymem, Aaron Mell, Acruid, Clyybber, Demetre Beroshvili, Flipp Syder, Javier Guardia Fernández, Julian Giebel, JustinTime, Matz05, Neith, Plasmaguy, TaralGit, Willhelm53, WlarusFromDaSpace, Ygg01, bloodrizer, keronshb, pointer-to-null, router, sewerpig*
```
Comment (ike709) [inline content/post/22-03-16-progress-report-31.md:162]:
```suggestion
*354 commits, authored by: metalgearsloth, Peptide90, Rane, Pancake, Kara D, ShadowCommander, Moony, Lamrr, mirrorcult, wrexbe, Pieter-Jan Briers, Vera Aguilera Puerto, LittleBuilderJane, Alex Evgrashin, Michael Phillips, Morbo, Sam Weaver, Paul Ritter, Daemon, Delete69, ElectroJr, DubiousDoggo, Júlio César Ueti, Paul, ScalyChimp, moonheart08, BubblegumBlue, Charlese2, DrSmugleaf, E F R, Fishfish458, T-Stalker, Zymem, Aaron Mell, Acruid, Clyybber, Demetre Beroshvili, Flipp Syder, Javier Guardia Fernández, Julian Giebel, JustinTime, Matz05, Neith, Plasmaguy, TaralGit, Willhelm53, WlarusFromDaSpace, Ygg01, bloodrizer, keronshb, pointer-to-null, router, sewerpig*
```

File: content/post/22-03-16-progress-report-31.md [hunk 7]
```diff
@@ -0,0 +1,165 @@
...
+
+### Patrons
+*Aaron Mell, Acruid, Alex Evgrashin, BubblegumBlue, Charlese2, Clyybber, Daemon, Delete69, Demetre Beroshvili, DrSmugleaf, E F R, ElectroJr, Fishfish458, Flipp Syder, Jack Fox, Julian Giebel, JustinTime, Júlio César Ueti, Kara D, Lamrr, LittleBuilderJane, Matz05, Michael Phillips, Moony, Morbo, Neith, Pancake, Paul, Peptide90, Pieter-Jan Briers, Plasmaguy, Rane, Sam Weaver, ScalyChimp, ShadowCommander, T-Stalker, TaralGit, Vera Aguilera Puerto, Willhelm53, WlarusFromDaSpace, Ygg01, Zymem, bloodrizer, keronshb, metalgearsloth, mirrorcult, pointer-to-null, router, sewerpig, wrexbe*
```
Comment (ike709) [inline content/post/22-03-16-progress-report-31.md:165]:
```suggestion
*Aaron Mell, Acruid, Alex Evgrashin, BubblegumBlue, Charlese2, Clyybber, Daemon, Delete69, Demetre Beroshvili, DrSmugleaf, E F R, ElectroJr, Fishfish458, Flipp Syder, DubiousDoggo, Julian Giebel, JustinTime, Júlio César Ueti, Kara D, Lamrr, LittleBuilderJane, Matz05, Michael Phillips, Moony, Morbo, Neith, Pancake, Paul, Peptide90, Pieter-Jan Briers, Plasmaguy, Rane, Sam Weaver, ScalyChimp, ShadowCommander, T-Stalker, TaralGit, Vera Aguilera Puerto, Willhelm53, WlarusFromDaSpace, Ygg01, Zymem, bloodrizer, keronshb, metalgearsloth, mirrorcult, pointer-to-null, router, sewerpig, wrexbe*
```

File: content/post/22-03-16-progress-report-31.md [hunk 8]
```diff
@@ -0,0 +1,165 @@
...
+
+Server hosts can now enable a whitelist using the `whitelist.enabled` cvar. Only players added with the command `whitelistadd <username>` will be able to join, and they can be removed with `whitelistremove <username>`.
+
+If you want to enable it mid-round, you can use the `kicknonwhitelisted` command to give every unwanted player the boot.
+
+{{< newsection >}}
+### Map Renderer
+*Contributed by DrSmugleaf and mirrorcult*
+
+`Content.MapRenderer` has been added as a new project that can turn a map file and render it as an image. It's extremely useful for development, updating the wiki, and more.
```
Comment (ike709) [inline content/post/22-03-16-progress-report-31.md:155]:
```suggestion
`Content.MapRenderer` has been added as a new project that can take a map file and render it as an image. It's extremely useful for development, updating the wiki, and more.
```

Comment (ike709) [inline content/post/22-03-16-progress-report-31.md:155]:
```suggestion
`Content.MapRenderer` has been added as a new project that can take a map file and render it as an image. It's extremely useful for development, updating the wiki, and more.
```

File: content/post/22-03-16-progress-report-31.md [hunk 9]
```diff
@@ -0,0 +1,165 @@
...
+### Server Whitelist
+*Contributed by mirrorcult*
+
+Server hosts can now enable a whitelist using the `whitelist.enabled` cvar. Only players added with the command `whitelistadd <username>` will be able to join, and they can be removed with `whitelistremove <username>`.
+
+If you want to enable it mid-round, you can use the `kicknonwhitelisted` command to give every unwanted player the boot.
+
+{{< newsection >}}
+### Map Renderer
+*Contributed by DrSmugleaf and mirrorcult*
```
Comment (ike709) [inline content/post/22-03-16-progress-report-31.md:153]:
```suggestion
*Contributed by DrSmugleaf, moony, and mirrorcult*
```

File: content/post/22-03-16-progress-report-31.md [hunk 10]
```diff
@@ -0,0 +1,165 @@
...
+*Contributed by DrSmugleaf, moony, and mirrorcult*
+
+`Content.MapRenderer` has been added as a new project that can take a map file and render it as an image. It's extremely useful for development, updating the wiki, and more.
+
+{{< imgw "images/post/pr_31/bagelrender" >}}
+
+{{< newsection >}}
+## Credits
+### Contributors Since Last Progress Report
+*354 commits, authored by: metalgearsloth, Peptide90, Rane, Pancake, Kara D, ShadowCommander, Moony, Lamrr, mirrorcult, wrexbe, Pieter-Jan Briers, Vera Aguilera Puerto, LittleBuilderJane, Alex Evgrashin, Michael Phillips, Morbo, Sam Weaver, Paul Ritter, Daemon, Delete69, ElectroJr, DubiousDoggo, Júlio César Ueti, Paul, ScalyChimp, moonheart08, BubblegumBlue, Charlese2, DrSmugleaf, E F R, Fishfish458, T-Stalker, Zymem, Aaron Mell, Acruid, Clyybber, Demetre Beroshvili, Flipp Syder, Javier Guardia Fernández, Julian Giebel, JustinTime, Matz05, Neith, Plasmaguy, TaralGit, Willhelm53, WlarusFromDaSpace, Ygg01, bloodrizer, keronshb, pointer-to-null, router, sewerpig*
```
Comment (DrSmugleaf) [inline content/post/22-03-16-progress-report-31.md:162]: Duplicate names.

```suggestion
*354 commits, authored by: metalgearsloth, Peptide90, Rane, Pancake, Kara D, ShadowCommander, Moony, Lamrr, mirrorcult, wrexbe, Pieter-Jan Briers, Vera Aguilera Puerto, LittleBuilderJane, Alex Evgrashin, Michael Phillips, Morbo, Sam Weaver, Daemon, Delete69, ElectroJr, DubiousDoggo, Júlio César Ueti, Paul, ScalyChimp, BubblegumBlue, Charlese2, DrSmugleaf, E F R, Fishfish458, T-Stalker, Zymem, Aaron Mell, Acruid, Clyybber, Demetre Beroshvili, Flipp Syder, Julian Giebel, JustinTime, Matz05, Neith, Plasmaguy, TaralGit, Willhelm53, WlarusFromDaSpace, Ygg01, bloodrizer, keronshb, pointer-to-null, router, sewerpig*
```

**Review by DrSmugleaf — CHANGES_REQUESTED** [review]

File: content/post/22-03-16-progress-report-31.md [hunk 11]
```diff
@@ -0,0 +1,165 @@
...
+
+### Patrons
+*Aaron Mell, Acruid, Alex Evgrashin, BubblegumBlue, Charlese2, Clyybber, Daemon, Delete69, Demetre Beroshvili, DrSmugleaf, E F R, ElectroJr, Fishfish458, Flipp Syder, DubiousDoggo, Julian Giebel, JustinTime, Júlio César Ueti, Kara D, Lamrr, LittleBuilderJane, Matz05, Michael Phillips, Moony, Morbo, Neith, Pancake, Paul, Peptide90, Pieter-Jan Briers, Plasmaguy, Rane, Sam Weaver, ScalyChimp, ShadowCommander, T-Stalker, TaralGit, Vera Aguilera Puerto, Willhelm53, WlarusFromDaSpace, Ygg01, Zymem, bloodrizer, keronshb, metalgearsloth, mirrorcult, pointer-to-null, router, sewerpig, wrexbe*
```
Comment (DrSmugleaf) [inline content/post/22-03-16-progress-report-31.md:165]: List of Patrons is wrong, I'm not one of them and Tomeno is missing for example.

**Review by DrSmugleaf — CHANGES_REQUESTED** [review]

File: content/post/22-03-16-progress-report-31.md (hunk 11 above)
Comment (DrSmugleaf) [inline content/post/22-03-16-progress-report-31.md:165]: Correct list:

```suggestion
*Phillip Inman, Zakanater 19, Jex N, Peter "Azmond" Newhouse, Seija, Tom Cruize, Corpus Inc., Hamcha, Oxyclean114, JustanAnus, NorthwoodTodd, Collin R Terrell, Cormos Lemming, grhmhome, Spiffyfishface, Callum Tubrett, Hydration, John Edward Hamilton Barchard, Wrexbe, Frege Beach, Austin Nelson, AquaDraco, Miniwoffer, hh, Nicholas Hillblom, Florian, Viridian, Daskata, Blaise M., Jack Rose, The Hateful Flesh, Will M., spinnermaster, Koyki, Valinov, Dan Griskardian, Robin Rottstock, Altana, Durp, Joshington Awesomahee, Eric VW, Evan Armstrong, Mono, Ethan Keller, DramaBuns, KrystalDisc, Darren Brady, Zandario, Anthony Fleck, vifs_vestige, Bobberunio, CPM311, ThatGuyGW, Farewell Fire, Daniel Thompson, Tomeno*
```

**Review by DrSmugleaf — CHANGES_REQUESTED** [review]

File: content/post/22-03-16-progress-report-31.md (hunk 11 above)
Comment (DrSmugleaf) [reply content/post/22-03-16-progress-report-31.md]: Done in [`d33a7e6` (#27)](https://github.com/space-wizards/website-content/pull/27/commits/d33a7e6311c71bc1065c548a98b3b044ea77e021)

**Review by DrSmugleaf — APPROVED** [review]

File: content/post/22-03-16-progress-report-31.md [hunk 12]
```diff
@@ -0,0 +1,165 @@
...
+*Contributed by DrSmugleaf, moony, and mirrorcult*
+
+`Content.MapRenderer` has been added as a new project that can take a map file and render it as an image. It's extremely useful for development, updating the wiki, and more.
+
+{{< imgw "images/post/pr_31/bagelrender" >}}
+
+{{< newsection >}}
+## Credits
+### Contributors Since Last Progress Report
+*354 commits, authored by: metalgearsloth, Peptide90, Rane, Pancake, Kara D, ShadowCommander, Moony, Lamrr, mirrorcult, wrexbe, Pieter-Jan Briers, Vera Aguilera Puerto, LittleBuilderJane, Alex Evgrashin, Michael Phillips, Morbo, Sam Weaver, Daemon, Delete69, ElectroJr, DubiousDoggo, Júlio César Ueti, Paul, ScalyChimp, BubblegumBlue, Charlese2, DrSmugleaf, E F R, Fishfish458, T-Stalker, Zymem, Aaron Mell, Acruid, Clyybber, Demetre Beroshvili, Flipp Syder, Julian Giebel, JustinTime, Matz05, Neith, Plasmaguy, TaralGit, Willhelm53, WlarusFromDaSpace, Ygg01, bloodrizer, keronshb, pointer-to-null, router, sewerpig*
```
Comment (mirrorcult) [inline content/post/22-03-16-progress-report-31.md:162]:
```suggestion
*354 commits, authored by: metalgearsloth, Peptide90, Rane, Pancake, ShadowCommander, Moony, Lamrr, mirrorcult, wrexbe, Pieter-Jan Briers, Vera Aguilera Puerto, LittleBuilderJane, Alex Evgrashin, Michael Phillips, Morbo, Sam Weaver, Daemon, Delete69, ElectroJr, DubiousDoggo, Júlio César Ueti, Paul, ScalyChimp, BubblegumBlue, Charlese2, DrSmugleaf, E F R, Fishfish458, T-Stalker, Zymem, Aaron Mell, Acruid, Clyybber, Demetre Beroshvili, Flipp Syder, Julian Giebel, JustinTime, Matz05, Neith, Plasmaguy, TaralGit, Willhelm53, WlarusFromDaSpace, Ygg01, bloodrizer, keronshb, pointer-to-null, router, sewerpig*
```

**Review by mirrorcult — CHANGES_REQUESTED** [review]

**Review by moonheart08 — APPROVED** [review]

**Review by mirrorcult — APPROVED** [review]

**Pull request merged**